app/src/main/java/io/rafaelribeiro/spendless/di/DatabaseModule.kt

```diff
@@ -17,11 +17,9 @@ object DatabaseModule {
     @Provides
     @Singleton
     fun provideSpendLessDatabase(@ApplicationContext context: Context): SpendLessDatabase {
-        return Room.databaseBuilder(
-            context,
-            SpendLessDatabase::class.java,
-            SpendLessDatabase.DATABASE_NAME
-        ).build()
+        return Room.databaseBuilder(context, SpendLessDatabase::class.java, SpendLessDatabase.DATABASE_NAME)
+            .fallbackToDestructiveMigration()
```

Comment:
Ideally, we should create a migration script, but for the sake of time and because there are no end users using the app yet, I am just destroying the database and recreating it again after migration.